cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2963,6 +2978,87 @@ String getInternalServiceHostname(String serviceName)    {
             return withVoid(Future.succeededFuture());
         }
 
+        Future<ReconciliationState> getJmxTransDescription() {
+            Promise<ReconciliationState> promise = Promise.promise();
+
+            vertx.createSharedWorkerExecutor("kubernetes-ops-pool").<ReconciliationState>executeBlocking(
```

Comment:
Is there any reason why this is blocking? I do not think we have any blocking code here, or?

Reply:
I think that this was copied from a previous function so I will take this out and follow a more up to date example :)